src/main/kotlin/io/polygon/kotlin/sdk/rest/Trades.kt

```diff
@@ -65,8 +65,11 @@ data class Trade(
     val conditions: List<Int>? = null,
     val correction: Int? = null,
     val exchange: Int? = null,
+    @SerialName("id") val tradeID: String? = null,
+    @SerialName("last_updated") val lastUpdated: Long? = null,
     @SerialName("participant_timestamp") val participantTimestamp: Long? = null,
     val price: Double? = null,
     @SerialName("sip_timestamp") val sipTimestamp: Long? = null,
     val size: Double? = null,
+    @SerialName("timeframe") val timeframe: String? = null,
```

Comment:
I don't think we should add this or `last_updated` to this trade model only for snapshot. They're different models internally as well so I think it'd be fine to copy this over into Snapshots.kt and call it something like `SnapshotTrade`